### PR TITLE
[Task 1869679] add eclipse store

### PR DESCRIPTION
**Pull request by andxu**
[AB#1869679](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1869679)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/store/EclipseStore.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @Override
+    public void setProperty(@Nullable String service, @NotNull String key, @Nullable String value) {
+    	if (value == null) {
+    		node.remove(combineKey(service, key));
+    		return;
+    		
+    	}
+        node.put(combineKey(service, key), value);
+        try {
+			node.flush();
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/store/EclipseStore.java:43]: check style

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/store/EclipseStore.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import com.microsoft.azure.toolkit.lib.common.exception.AzureToolkitRuntimeException;
+import com.microsoft.azuretools.core.Activator;
+
+public class EclipseStore implements IIdeStore {
+    private IEclipsePreferences node = ConfigurationScope.INSTANCE.getNode(Activator.PLUGIN_ID);
+
+    @Nullable
+    @Override
+    public String getProperty(@Nullable String service, @NotNull String key) {
+        return node.get(combineKey(service, key), "");
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/store/EclipseStore.java:25]: using null as the default value to keep consistent with security store?

**Review by Flanker32 — APPROVED** [review]
Please resolve the checkstyle issues

**Comment by andxu** [comment]
Will resolve it in later PR

**Pull request merged**